[2023-03-04T15:59:52.964+0100] {processor.py:154} INFO - Started process (PID=64190) to work on /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T15:59:52.972+0100] {processor.py:756} INFO - Processing file /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-04T15:59:52.972+0100] {logging_mixin.py:137} INFO - [2023-03-04T15:59:52.972+0100] {dagbag.py:537} INFO - Filling up the DagBag from /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T15:59:53.033+0100] {logging_mixin.py:137} INFO - [2023-03-04T15:59:53.032+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-04T15:59:53.033+0100] {logging_mixin.py:137} INFO - [2023-03-04T15:59:53.033+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-04T15:59:53.033+0100] {processor.py:768} WARNING - No viable dags retrieved from /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T15:59:53.159+0100] {processor.py:176} INFO - Processing /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.196 seconds
[2023-03-04T16:01:10.522+0100] {processor.py:154} INFO - Started process (PID=64244) to work on /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T16:01:10.527+0100] {processor.py:756} INFO - Processing file /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-04T16:01:10.528+0100] {logging_mixin.py:137} INFO - [2023-03-04T16:01:10.528+0100] {dagbag.py:537} INFO - Filling up the DagBag from /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T16:01:10.574+0100] {logging_mixin.py:137} INFO - [2023-03-04T16:01:10.573+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-04T16:01:10.574+0100] {logging_mixin.py:137} INFO - [2023-03-04T16:01:10.574+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-04T16:01:10.574+0100] {processor.py:768} WARNING - No viable dags retrieved from /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T16:01:10.676+0100] {processor.py:176} INFO - Processing /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.154 seconds
[2023-03-04T16:02:42.743+0100] {processor.py:154} INFO - Started process (PID=64308) to work on /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T16:02:42.749+0100] {processor.py:756} INFO - Processing file /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-04T16:02:42.749+0100] {logging_mixin.py:137} INFO - [2023-03-04T16:02:42.749+0100] {dagbag.py:537} INFO - Filling up the DagBag from /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T16:02:42.804+0100] {logging_mixin.py:137} INFO - [2023-03-04T16:02:42.804+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-04T16:02:42.804+0100] {logging_mixin.py:137} INFO - [2023-03-04T16:02:42.804+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-04T16:02:42.804+0100] {processor.py:768} WARNING - No viable dags retrieved from /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T16:02:42.914+0100] {processor.py:176} INFO - Processing /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.172 seconds
[2023-03-04T16:04:14.116+0100] {processor.py:154} INFO - Started process (PID=64388) to work on /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T16:04:14.121+0100] {processor.py:756} INFO - Processing file /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-03-04T16:04:14.122+0100] {logging_mixin.py:137} INFO - [2023-03-04T16:04:14.122+0100] {dagbag.py:537} INFO - Filling up the DagBag from /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T16:04:14.166+0100] {logging_mixin.py:137} INFO - [2023-03-04T16:04:14.165+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-03-04T16:04:14.166+0100] {logging_mixin.py:137} INFO - [2023-03-04T16:04:14.166+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-03-04T16:04:14.166+0100] {processor.py:768} WARNING - No viable dags retrieved from /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-03-04T16:04:14.265+0100] {processor.py:176} INFO - Processing /Users/uab910/dev/nnam-poc/pyenv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.149 seconds
